sdk/init.go

```diff
@@ -224,7 +224,7 @@ func New(
 	}
 
 	var namespace = "Mock"
-	if chainID.Int64() == constants.MainnetChainID {
+	if chainID.Int64() == constants.MainnetChainID || chainID.Int64() == constants.CalibnetChainID {
```

Comment:
I think we should just scrap mock all together. Just delete this whole block:

`namespace := "ADO"` and no conditional checks.